src/OrchardCore.Modules/OrchardCore.Alias/Migrations.cs

```diff
@@ -24,10 +24,10 @@ public async Task<int> CreateAsync()
                 .WithDescription("Provides a way to define custom aliases for content items."));
 
             await SchemaBuilder.CreateMapIndexTableAsync<AliasPartIndex>(table => table
-                .Column<string>("Alias", col => col.WithLength(AliasPart.MaxAliasLength))
-                .Column<string>("ContentItemId", c => c.WithLength(26))
-                .Column<bool>("Latest", c => c.WithDefault(false))
-                .Column<bool>("Published", c => c.WithDefault(true))
+                .Column<string>("Alias", column => column.WithLength(AliasPart.MaxAliasLength))
+                .Column<string>("ContentItemId", column => column.NotNull().WithLength(26))
```

Comment:
I'm glad that somebody else understands my aversion of single-letter lambda parameters :D.